lib/improver/tests/ensemble_calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py

```diff
@@ -708,8 +744,7 @@ def test_coefficient_values_for_gaussian_distribution_realizations(self):
         if statsmodels_found:
             data = [-0.00114, -0.00006, 1.00037, -0.00196, 0.99999, -0.00315]
         else:
-            data = [4.30804737e-02, 1.39042785e+00, 8.99047025e-04,
-                    2.02661310e-01, 9.27197381e-01, 3.17407626e-01]
+            data = [0.9287, -0.0567, -0.00926, 0.16589, 0.74932, 0.64316]
```

Comment:
These numbers have also changed a lot, can we have a chat about what they mean?

Reply:
With 1000 iterations:
```
-7.21049e-07, 1.70712e-07, 0.00516975, -0.21115, 0.607316, 0.765887
```